master_preset.xml

```diff
@@ -1100,11 +1100,15 @@
         <preset_link preset_name="Physical restrictions"/>
         <preset_link preset_name="Other mode restrictions"/>
     </chunk>
+    <chunk id="dog">
+        <combo key="dog" text="Dog" values="yes,leashed,unleashed,designated,permissive,private,no"
+            display_values="Yes,Leashed,Unleashed,Designated,Permissive,Private,No" values_sort="false" values_context="access" match="key" />
+    </chunk>
     <chunk id="path_access_restrictions">
         <optional>
             <combo key="horse" text="Horse" values="yes,official,designated,permissive,destination,delivery,private,permit,no"
                 display_values="Yes,Official,Designated,Permissive,Destination,Delivery,Private,Permit required,No" values_context="access" match="key" values_sort="false"/>
-            <combo key="dog" text="Dog" values="yes,leashed,unleashed,official,designated,permissive,private,permit,no" 
+            <reference ref="dog" />
                 display_values="Yes,Leashed,Unleashed,Official,Designated,Permissive,Private,Permit required,No" values_sort="false" values_context="access"  match="key" />
```

Comment:
This is causing the validation error, you probably just forgot to delete line 1112

Reply:
Indeed. Good catch! Sorry for the hassle!